app/models/user.rb

```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
-  has_many :user_events
+  has_many :user_events, dependent: :destroy
```

Comment:
I'm interested to learn about what dependent: :destroy is doing here.

Reply:
By adding the `dependent: :destroy` option to the has_many :through associations, the User record will be removed when UserEvents row is destroyed.

But I just realized that I am using it wrong here and will submit a PR fix shortly.